packages/ujson/meta.yaml

```
@@ -0,0 +1,129 @@
+package:
+  name: ujson
+  version: 5.10.0
+  top-level:
+    - ujson
+source:
+  url: https://files.pythonhosted.org/packages/f0/00/3110fd566786bfa542adb7932d62035e0c0ef662a8ff6544b6643b3d6fd7/ujson-5.10.0.tar.gz
```

With predictable URLs from PyPI, this can be simplified as follows:

```suggestion
  url: https://files.pythonhosted.org/packages/source/u/ujson/ujson-5.10.0.tar.gz
```